README_Hindi.md

```diff
@@ -151,10 +198,26 @@ Libraries तथा Tools
 -   [vearch](https://github.com/vearch/vearch)
 -   [PyNNDescent](https://github.com/lmcinnes/pynndescent)
 -   [pgANN](https://github.com/netrasys/pgANN)
+-   [Tensorflow Similarity](https://github.com/tensorflow/similarity)
+-   [opensemanticsearch.org](https://www.opensemanticsearch.org/)
+-   [GPT3 Semantic Search](https://gpt3demo.com/category/semantic-search)
+-   [searchy](https://github.com/lubianat/searchy)
+-   [txtai](https://github.com/neuml/txtai)
+-   [HyperTag](https://github.com/Ravn-Tech/HyperTag)
+-   [vectorai](https://github.com/vector-ai/vectorai)
+-   [embeddinghub](https://github.com/featureform/embeddinghub)
+-   [AquilaDb](https://github.com/Aquila-Network/AquilaDB)
 
 डेटासेट
 -------
 
--   [सिमेंटिक टेक्स्ट समानता डेटासेट
+- [सिमेंटिक टेक्स्ट समानता डेटासेट
     हब](https://github.com/brmson/dataset-sts)
+- [फेसबुक एआई छवि समानता चुनौती](https://www.drivendata.org/competitions/79/competition-image-similarity-1-dev/?fbclid=IwAR31vRV0EdxRdrxtPy12neZtBJQ0H9qdLHm8Wl2DjHY09PtQdn1nEEIJVUo)
+- [WIT: विकिपीडिया-आधारित छवि पाठ डेटासेट](https://github.com/google-research-datasets/wit)
+
+
+Milestones
+-------
 
+- कार्य सूची के लिए [परियोजना बोर्ड](https://github.com/Agrover112/awesome-semantic-search/projects/1) पर एक नज़र डालें ताकि किसी भी खुले मुद्दे में योगदान किया जा सके।
```

Comment:
@WebShivam  Are you using Google Translate or doing this yourself? If you are using Google Translate I would recommend you to use bing translate / DuckDuckGos's translation feature since they probably are using GPT-3 and results in much natural translations.

Reply:
Thanks for the suggestion @Agrover112 :) I would surely check them out. They can be really useful 👍 

Currently, I'm using the combinations of both Google and Bing's Translations and choosing the one which fits the best meaning.